CL/cl_ext.h

```diff
@@ -2441,6 +2441,11 @@ typedef struct _cl_queue_family_properties_intel {
 #define CL_QUEUE_CAPABILITY_BARRIER_INTEL                   (1 << 25)
 #define CL_QUEUE_CAPABILITY_KERNEL_INTEL                    (1 << 26)
 
+/* cl_intel_queue_no_sync_operations */
+#define cl_intel_queue_no_sync_operations 1
+
+#define CL_QUEUE_NO_SYNC_OPERATIONS_INTEL                   (1 << 4)    
```

Comment:
How much trouble would it be to choose a different bit number than "4" for this enum?

While there's no requirement to do this, for other bitfields (e.g. cl_mem_flags, cl_command_queue_properties) we've assigned bits for vendor extensions to higher bits (e.g. 31 on down), leaving the lower bits free for standard functionality.  I'd recommend continuing this convention if at all possible.

Reply:
Sure, no problem.